ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetLogs.java

```diff
@@ -107,12 +114,14 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
           .addArgument(requestContext.getRequest())
           .setCause(ex.get())
           .log();
-      if (ex.get() instanceof IllegalArgumentException) {
+      if (ex.get() instanceof InvalidJsonRpcParameters) {
         return new JsonRpcErrorResponse(
-            requestContext.getRequest().getId(), RpcErrorType.EXCEEDS_RPC_MAX_BLOCK_RANGE);
+            requestContext.getRequest().getId(),
+            ((InvalidJsonRpcParameters) ex.get()).getRpcErrorType());
+      } else {
+        LOG.error("Unexpected exception: {}", ex.get(), ex.get());
+        // TODO: wrap and throw? Something obviously went wrong
```

Comment:
think we should throw here

Reply:
Yep, handled in a previous branch and now merged to main.